content/docs/tasks/security/health-check/index.md

```diff
@@ -22,14 +22,11 @@ this feature is not needed if the production setup is not using the
 
 ## Before you begin
 
-To complete this task, you can install Istio using one of the following paths:
+To complete this task, you can install Istio using **one of** the following paths:
 
 * To setup Istio without using Helm, follow the instructions in the
-  [Kubernetes installation guide](/docs/setup/kubernetes/install/kubernetes/). Remember to enable global mutual TLS with:
-
-    {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
-    {{< /text >}}
+  [installation guide](/docs/setup/kubernetes/install/kubernetes/).
+  Install Istio with the **strict mTLS mode** enabled.
 
 * Use [Helm](/docs/setup/kubernetes/install/helm/) to setup Istio and set the `global.mtls.enabled` flag to `true`.
```

Comment:
for some reason, I thought global.mtls.enabled was removed.

Reply:
turns out global.mtls.enabled was deprecated - so this will need more fleshing out in the future.